QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -64,8 +64,8 @@ public void event() {
     public static final EventOutput setBottom = EventMixing.combine(raising.getSetFalseEvent(), lowering.getSetTrueEvent());
     public static final EventOutput stop = BooleanMixing.getSetEvent(BooleanMixing.combine(raising, lowering), false);
 
-    private static final BooleanStatus atTopStatus = new BooleanStatus();
-    private static final BooleanStatus atBottomStatus = new BooleanStatus();
+    public static final BooleanStatus atTopStatus = new BooleanStatus();
+    public static final BooleanStatus atBottomStatus = new BooleanStatus();
```

Comment:
Alternatively, I'm missing something very obvious, but I don't see why you can't just read these through `atTop`